src/lib/config.ts

```diff
@@ -0,0 +1,75 @@
+import path from 'path'
+import fs from 'fs'
+import { WebConfigSchema, WebStaticConfigSchema } from '../types.js'
+
+export const DEFAULT_WEB_STATIC_CONFIG: WebStaticConfigSchema = [
+    {
+        "name": "lambdatest-home-page",
+        "url": "https://www.lambdatest.com",
+        "waitForTimeout": 1000
+    },
+    {
+        "name": "example-page",
+        "url": "https://example.com/"
+    }
+]
+
+export const DEFAULT_WEB_CONFIG: WebConfigSchema = {
+    web: {
+        browsers: [
+            'chrome',
+            'firefox',
+            'safari',
+            'edge'
+        ],
+        viewports: [
+            [1920, 1080],
+            [1366, 768],
+            [360, 640],
+        ]
+    }
+};
+
+export function createWebConfig(filepath: string) {
+    // default filepath
+    filepath = filepath || 'smartui-web.json';
+    let filetype = path.extname(filepath);
+    if (filetype != '.json') {
+        console.log('Error: Config file must have .json extension');
+        return
+    }
+
+    // verify the file does not already exist
+    if (fs.existsSync(filepath)) {
+        console.log(`Error: SmartUI Web Config already exists: ${filepath}`);
+        console.log(`To create a new file, please specify the file name like: 'smartui config:create-web webConfig.json'`);
```

Comment:
@pinanks Please add logger instead of console logs

Reply:
Will have to use console.log for now (DOT-2260)